pkg/cmd/workspace/start.go

```diff
@@ -30,10 +29,9 @@ var codeFlag bool
 var StartCmd = &cobra.Command{
 	Use:     "start [WORKSPACE]",
```

Comment:
Usage should be `start [WORKSPACE]...` since we support multi-start with args now (applies to stop as well)